sdk/avs/arm-avs/CHANGELOG.md

```diff
@@ -1,15 +1,25 @@
 # Release History
-
-## 3.2.1 (2023-01-11)
-
+    
+## 4.0.0 (2023-08-10)
+    
 **Features**
- 
-  - Exposes `getContinuationToken` helper function to extract continuation token
 
-**Bugs Fixed**
- 
-  - A series of small bug fixs relevant to authentication and apiVersion policy
+  - Added Interface ErrorDetail
+  - Interface ErrorResponse has a new optional parameter error
+  - Interface PrivateCloud has a new optional parameter extendedNetworkBlocks
+  - Interface PrivateCloudUpdate has a new optional parameter extendedNetworkBlocks
+  - Interface PrivateCloudUpdateProperties has a new optional parameter extendedNetworkBlocks
+  - Added function getContinuationToken
+
+**Breaking Changes**
 
+  - Interface ErrorResponse no longer has parameter additionalInfo
```

Comment:
Could you help check the breaking pr?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/24490

Reply:
We can only accept this breaking.